docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Reduce database size using Bonsai Tries
+sidebar_position: 12
+description: Enable this feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To decrease the database size using the [Bonsai Trie](../concepts/data-storage-formats#bonsai-tries) data storage policy in Besu, enable the early access feature `--Xbonsai-limit-trie-logs-enabled`. 
+When enabled, this feature can reduce database growth by more than 3GB each week on mainnet.
+
+## Limit Trie Logs for Bonsai
+
+:::caution
+
+The following commands are examples. Before executing these example commands on your node, modify them to apply to your node's configuration.
+
+:::
+
+1. Add the `--Xbonsai-limit-trie-logs-enabled` option to the [Besu configuration file](configuration-file).
+1. Stop Besu. If Besu is running in a terminal, you can usually stop it by pressing `Ctrl+C`.
+1. (optional) Run the Besu client. Specify the Bonsai Trie data storage format and a data directory for the pruned trie logs:
+
+    ```bash title="Specify Bonsai Trie for pruned logs"
+    sudo /usr/local/bin/besu/bin/besu --data-storage-format=BONSAI --data-path=/var/lib/besu --sync-mode=X_SNAP storage x-trie-log prune
+    ```
+1. Restart Besu. You can restart Besu by navigating to the directory where your Besu executable is located and running the same command you used to start it initially. 
+If you are using a `systemd` service file, as recommended by [CoinCashew](https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet/part-i-installation/step-3-installing-execution-client/besu) 
+and [Somer](https://someresat.medium.com/guide-to-staking-on-ethereum-ubuntu-teku-f09ecd9ef2ee), ensure you execute `sudo systemctl daemon-reload`.
+1. Look for `Limit trie logs enabled: retention: 512; prune window: 30000` in your Besu configuration printout at startup.
+
+## Restart Besu
+
+When Besu restarts, it will prune and remove unnecessary data, one block at a time. 
```

Comment:
```suggestion
Once Besu is started with `--Xbonsai-limit-trie-logs-enabled`, it will continuously prune, removing unnecessary Trie Log data one block at a time. 
```